packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -9,6 +9,7 @@ import algosdk, {
 	getApplicationAddress,
 	isValidAddress,
 	modelsv2,
+	SignedTransaction,
 	verifyBytes,
 } from "algosdk";
 import { setSendTransactionHeaders } from "algosdk/dist/types/src/client/v2/algod/sendRawTransaction";
```

Comment:
I think we should import it from algosdk/types( should not from algosdk/dist). Please check it!